test/test_config.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+from app import app, db, models
+
+class TestDevelopmentConfig(unittest.TestCase):
+    def create_app(self):
+        app.config.from_object('Bucketlist-Server.config.DevelopmentConfig')
+        return app
+
+    def test_app_is_development(self):
+        self.assertTrue(app.config['DEBUG'] is True)
+        self.assertFalse(current_app is None)
+        self.assertTrue(
+            app.config['SQLALCHEMY_DATABASE_URI'] == 'postgresql://postgres:@localhost/flask_jwt_auth'
+        )
+
+
+class TestTestingConfig(unittest.TestCase):
+    def create_app(self):
+        app.config.from_object('Bucketlist-Server.config.TestingConfig')
+        return app
+
+    def test_app_is_testing(self):
+        self.assertTrue(app.config['DEBUG'])
+        self.assertTrue(
+            app.config['SQLALCHEMY_DATABASE_URI'] == 'postgresql://postgres:@localhost/flask_jwt_auth_test'
+        )
+def test_app_is_development(self):
+    self.assertFalse(app.config['SECRET_KEY'] is 'my_precious')
+    self.assertTrue(app.config['DEBUG'] is True)
```

Comment:
Since the value of `app.config['DEBUG']` is True you don't need the statement `is True`.
You can refactor your code:
`self.assertTrue(app.config['DEBUG'])`